docs/maps/geoheatmap-series.md

```diff
@@ -0,0 +1,37 @@
+GeoHeatMap Series
+=================
+
+A GeoHeatMap series is a graphical representation of data on different map projections where the individual values contained in a matrix are represented as colors.
+
+<iframe style="width: 100%; height: 520px; border: none;" src="https://highcharts.com/samples/embed/maps/demo/geoheatmap" allow="fullscreen"></iframe>
+
+GeoHeatMap similar to regular [HeatMap](https://www.highcharts.com/docs/chart-and-series-types/heatmap) is commonly used to visualize hot spots within data sets and to show patterns or correlations but this time the matrix respects the actual coordinates. Due to their compact nature, they are often used with large sets of data.
+
+### Setting up the geo heat map series
```

Comment:
In this article, the terms "GeoHeatMap", "geo heat map" and "geohetmap" are used inconsistently. Please pick one and use it consistently. I think GeoHeatMap is best. Unless when speaking of the series type `geoheatmap`.